kv/kvserver: TestRangefeedCheckpointsRecoverFromLeaseExpiration failed

kv/kvserver.TestRangefeedCheckpointsRecoverFromLeaseExpiration [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4146213&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4146213&tab=artifacts#/) on master @ [28944f9adf4df64a7e059a98b5b90adcc4b20d1d](https://github.com/cockroachdb/cockroach/commits/28944f9adf4df64a7e059a98b5b90adcc4b20d1d):


```
=== RUN   TestRangefeedCheckpointsRecoverFromLeaseExpiration
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRangefeedCheckpointsRecoverFromLeaseExpiration1528879470
    test_log_scope.go:80: use -show-logs to present logs inline
    replica_rangefeed_test.go:1049: 
        	Error Trace:	replica_rangefeed_test.go:1049
        	Error:      	Received unexpected error:
        	            	request not evaluated locally; evaluated by s1 instead of local s2
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/server.(*TestServer).GetRangeLease
        	            	  | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/testserver.go:1315
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/testcluster.(*TestCluster).FindRangeLeaseEx
        	            	  | 	/go/src/github.com/cockroachdb/cockroach/pkg/testutils/testcluster/testcluster.go:1046
        	            	  | github.com/cockroachdb/cockroach/pkg/kv/kvserver_test.TestRangefeedCheckpointsRecoverFromLeaseExpiration
        	            	  | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_rangefeed_test.go:1048
        	            	  | testing.tRunner
        	            	  | 	/usr/local/go/src/testing/testing.go:1259
        	            	  | runtime.goexit
        	            	  | 	/usr/local/go/src/runtime/asm_amd64.s:1581
        	            	Wraps: (2) request not evaluated locally; evaluated by s1 instead of local s2
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestRangefeedCheckpointsRecoverFromLeaseExpiration
    panic.go:642: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRangefeedCheckpointsRecoverFromLeaseExpiration1528879470
--- FAIL: TestRangefeedCheckpointsRecoverFromLeaseExpiration (0.92s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
/cc @cockroachdb/kv
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRangefeedCheckpointsRecoverFromLeaseExpiration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

kv/kvserver.TestRangefeedCheckpointsRecoverFromLeaseExpiration [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4223754&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4223754&tab=artifacts#/) on master @ [659fe193978a65187fcce81e985e14f9e2de9606](https://github.com/cockroachdb/cockroach/commits/659fe193978a65187fcce81e985e14f9e2de9606):


```
=== RUN   TestRangefeedCheckpointsRecoverFromLeaseExpiration
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRangefeedCheckpointsRecoverFromLeaseExpiration4208452139
    test_log_scope.go:80: use -show-logs to present logs inline
    replica_rangefeed_test.go:1049: 
        	Error Trace:	replica_rangefeed_test.go:1049
        	Error:      	Received unexpected error:
        	            	request not evaluated locally; evaluated by s1 instead of local s2
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/server.(*TestServer).GetRangeLease
        	            	  | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/testserver.go:1323
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/testcluster.(*TestCluster).FindRangeLeaseEx
        	            	  | 	/go/src/github.com/cockroachdb/cockroach/pkg/testutils/testcluster/testcluster.go:1051
        	            	  | github.com/cockroachdb/cockroach/pkg/kv/kvserver_test.TestRangefeedCheckpointsRecoverFromLeaseExpiration
        	            	  | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_rangefeed_test.go:1048
        	            	  | testing.tRunner
        	            	  | 	/usr/local/go/src/testing/testing.go:1259
        	            	  | runtime.goexit
        	            	  | 	/usr/local/go/src/runtime/asm_amd64.s:1581
        	            	Wraps: (2) request not evaluated locally; evaluated by s1 instead of local s2
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestRangefeedCheckpointsRecoverFromLeaseExpiration
    panic.go:642: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestRangefeedCheckpointsRecoverFromLeaseExpiration4208452139
--- FAIL: TestRangefeedCheckpointsRecoverFromLeaseExpiration (1.40s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRangefeedCheckpointsRecoverFromLeaseExpiration.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/kv/kvserver/replica_rangefeed_test.go
+++ b/pkg/kv/kvserver/replica_rangefeed_test.go
@@ -1045,7 +1045,7 @@ func TestRangefeedCheckpointsRecoverFromLeaseExpiration(t *testing.T) {
 	require.Equal(t, int64(1), nudged)
 
 	// Check that n2 renewed its lease, like the test intended.
-	li, _, err := tc.FindRangeLeaseEx(ctx, desc, &n2Target)
+	li, _, err := tc.FindRangeLeaseEx(ctx, desc, nil)
 	require.NoError(t, err)
 	require.True(t, li.Current().OwnedBy(n2.GetFirstStoreID()))
 	require.Equal(t, int64(2), li.Current().Epoch)

